torch_glow/tests/nodes/size_test.py

```diff
@@ -19,3 +19,25 @@ def test_f(a):
         x = torch.zeros([4], dtype=torch.int32)
 
         jitVsGlow(test_f, x, expected_fused_ops={"aten::size"})
+
+    @unittest.skip(reason="not ready")
```

Comment:
What's the issue with this?

Reply:
@yinghai, the existing test_size_basic was marked as skip "not ready" and when enabled it fails with COMPILE_UNSUPPORTED_NODE_AFTER_OPTIMIZE. When running test_size_neg_dim, there was an "AssertionError: Expected aten::size to be fused" regardless of the negative index which appeared to me to be related to the preceding error and not related to the negative index so I marked the new test as skip "not ready" as well. The test_size_oob_neg_dim test can run because the index error is raised by pytorch before reaching glow.